### ISSUE TITLE
Doesn't work in iOS Safari and Chrome

### ISSUE THREAD

**Issue by mikereem** [GrapesJS/touch#1]
I was testing the demo site (index.html) in iOS 12.1.4 with Safari and Chrome and it doesn't work. I cannot select the text on the canvas, so I cannot access the context menu with the icons.
It works in a desktop Chrome with mobile emulation.

On my iOS/Safari and iOS/Chrome, I have tried the demo of DragDropTouch.js and it works fine.

The version of GrapesJS I was testing with: 0.14.52
The version of grapesjs-touch: 0.1.1
I was also using these plugins: 

grapesjs-blocks-basic 0.1.8
grapesjs-blocks-flexbox 0.1.1

But as I mentioned I have also tried the demo index.html from the grapesjs-touch source.

**Comment by mikereem** [comment]
Actually I found out that the drag and drop would work with this plugin (for example moving a new component to the canvas). But it is impossible to select anything on the canvas.

It is becase the touch devices doesn't fire a 'click' event, but a 'touchstart' and 'touchend'. I solved this issue by extending the SelectComponent.js in the grapesjs code. I have modified the toggleSelectComponent(enable) method by adding
methods[method](body, 'touchend', this.onClick);

This will invoke the onClick() handler method in case a touch device is used. However a computer might support both. In that case the onClick() is fired twice (once for the click and once for the touchend event). To handle this, I have added this line to the onClick() method:

e.preventDefault();

right after the e.stopPropagation();

I think this modification could be included in the main grapesjs project.
